src/graphics.rs

```diff
@@ -930,22 +930,30 @@ impl Context {
     }
 
     pub fn apply_uniforms<U>(&mut self, uniforms: &U) {
+        self.apply_uniforms_buffer(unsafe {
+            std::slice::from_raw_parts(std::mem::transmute(uniforms), std::mem::size_of::<U>())
```

Comment:
I am concerned about this line. Is everything works now like it works before? Is this transmute safe?

Reply:
Transmute removed.